### DISCUSSION TITLE
Change character height

### DISCUSSION THREAD

**Discussion by Hazmat616** [FakeFishGames/Barotrauma#12576] [Ideas and suggestions]
Allow us to change our height as part of character creation and have npcs have randomized height. Basically allow us to change our scale by say -10% to +10% and since a smaller hit box could be advantageous have it reduce your max vitality.

**Comment by SquidingTin** [comment]
This would probably break a lot of the game as the game suspects you to be a set height, if it was changed, you could fit though doors neatly, or if you were shorter, you could get into areas your not really meant to be

**Reply by bookslayer10** [reply]
You can already get through tiny gaps by swimming through them underwater, so this wouldn't affect cave navigation that much. I'm not sure what else you mean by "areas you are not supposed to be," there aren't any ships that have closed off sections like that.

It would be funny to play a tall character that hits their head on doorframes and needs to stoop in low rooms, and I could see people intentionally playing a short character to maneuver easier while repairing the submarine. If your character was taller you would be easier to hit, but you also would have longer melee range.

**Comment by Arctodites** [comment]
I could see this causing problems, like what SquidingTin says.  But one thing that makes Europan settlements feel a bit sterile and same-y is that there aren't any children running around.  Different human character body types (thin / fat, short / tall) would add diversity to the game and potentially open up interesting choices when taking on new crew members.  Seems like it would be a lot of work for the devs, though.

**Reply by Hazmat616** [reply]
Yeah it's funny how we have an event involving teenagers yet none are in game.

**Reply by jigardi** [reply]
> I could see this causing problems, like what SquidingTin says. But one thing that makes Europan settlements feel a bit sterile and same-y is that there aren't any children running around. Different human character body types (thin / fat, short / tall) would add diversity to the game and potentially open up interesting choices when taking on new crew members. Seems like it would be a lot of work for the devs, though.

This would be great for the creatures too - and help things feel fresh for longer.
